sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -158,6 +160,22 @@ public DataLakeDirectoryAsyncClient getDirectoryAsyncClient(String directoryName
             blockBlobAsyncClient);
     }
 
+    /**
+     * Creates a new DataLakeDirectoryAsyncClient object by concatenating "" to the end of
```

Comment:
Just so it stylizes the `""` differently.

```suggestion
     * Initializes a new DataLakeDirectoryAsyncClient object by concatenating {@code ""} to the end of
```